### ISSUE TITLE
[1.16.4] New step sounds dont play for some slab, wall, stair variants like Cobblestone

### ISSUE THREAD

**Issue by Inonedn** [OreCruncher/DynamicSurroundings#626]

**Comment by OreCruncher** [comment]
Yep - I have a bit of cleaning up to do.  Would also include all the new nether blocks.

**Comment by OreCruncher** [comment]
I pushed 1.16.4-4.0.1.2 that has a fix for default sound acoustic association as well as additional configs in support of the new MC blocks.  Check it out and let me know what you think.

**Comment by OreCruncher** [comment]
@Inonedn Have you had a chance to check out the latest releases?  If the issue is corrected I would like to resolve.  If there are specific vanilla sound acoustics that still aren't correct could you create separate tickets?

**Comment by The-Bernician-Lamb** [comment]
Hello,

I hope you don't mind me chiming in on this issue, but from my experience and all the blocks I've tested, it seems like all slabs, stairs and fences make the correct sounds (Both modded and vanilla blocks were tested). 

However I've noticed that walls of all types (both modded and vanilla) do not make any sound when walked on. (That includes with the latest version - 4.0.2.0).

There are also some vanilla blocks that I feel don't make the correct sounds when walked on for what material they are (Especially since most of these blocks got brand new sounds in vanilla 1.16)(Note: These concerns are more a matter of opinion than bugs) These blocks include: 

- Bone blocks
- Nether brick blocks and red nether brick blocks (all types)
- Basalt and polished basalt
- Netherrack
- Quartz bricks
- Crying obsidian
- Respawn Anchor
- Smithing table
- Shroomlight

(Let me know if you want me to make a separate issue for that last part)

**Comment by OreCruncher** [comment]
The new block sounds are on my list of things to work on.  I have to say they did a pretty good job with them.  From a footstep acoustic standpoint I think all I need to do is tweak pitch based on the type of move event (run, walk, landing, etc.).  BTW, the latest version on CurseForge has a new option where you can turn off DS sound acoustics and replace with the Vanilla ones.  You will still get the other sound effects such as armor and walking through things.

Off to see what is going on with walls.

**Comment by The-Bernician-Lamb** [comment]
Hi,

Just though that I'd give my thoughts on the latest update for 1.16.4 (4.0.3.2) as well as respond to your requests for the feedback that you posted on curseforge:

- Love the new sounds for the nether blocks! They fit really well! Some blocks that you may have missed are nether wart blocks, warped wart blocks, quartz bricks and lodestones. (I'm unsure if the current sounds for those blocks are intentional or not).
- Netherite armour sounds are perfect! No complaints there.
- Walls make sounds like they are supposed to now! The only thing left to do with them is to change the sounds for specific types of walls to fit better with their material. (Those being sandstone walls, red sandstone walls, nether brick walls and red nether brick walls).
- A bug I've noticed happening in this version is that when jumping or landing on certain blocks  such as lodestones, warped nylium, nylium they make sounds that sand makes instead of their specific material. This is just when jumping or landing on the block, all the other sounds for these blocks are fine.
- I have only been able to get Auroras to appear once or twice. I would recommend making them a bit more frequent (Unless this is intentional and they need specific criteria apart from biome to actually appear). When I did see them though, they looked great!

I hope this is helpful! Thanks for the amazing mod!

**Comment by OreCruncher** [comment]
* Quartz Bricks - AFAIKT they have the same SoundType as quartz.
* Walls - need to check on that.  They should be playing variants.  Could be a case I am missing.
* Sand on jump - it's possible.  For softer blocks sand or dirt is usually used as an accent for "wander" (just moving a little bit) or the "jump".  I will see if I can tune.
* The aurora is an always on thing at night if you are standing in a biome that spawns them.  I have noticed that the aurora is spawning in some strange locations in my Direwolf20 instance, which doesn't happen in my standard dev environment.  I think something is screwing around with the matrix stack before the aurora renders.

**Comment by OreCruncher** [comment]
Found the aurora problem.  You weren't seeing it because it was rendering in a strange place. lol.
